src/user/user.service.ts

```diff
@@ -60,7 +60,7 @@ export class UserService {
 
     if (!userFoundByEmail) {
       // if the email doesn't exists, throw a bad request exception
-      throw new HttpException("The user doesn't exists", HttpStatus.BAD_REQUEST);
+      throw new BadRequestException("The user doesn't exists")
```

Comment:
If the user does not exist, it should be a Not Found exception (status code 404)